### PR TITLE
Vite upgrade

### PR DESCRIPTION
**Pull request by mjrobison**
Adding in Vite and Vue3

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,5 @@
-NODE_ENV=development
-VUE_APP_API_ENDPOINT="https://api.mhacsports.com/"
-VUE_APP_CONTENT_API="https://api.mhacsports.com/api/v2/"
-VUE_APP_AWS_COOKIE_LOCATION="localhost"
+APP_ENV=development
+VITE_APP_API_ENDPOINT="https://api.mhacsports.com/"
+VITE_APP_CONTENT_API="https://api.mhacsports.com/api/v2/"
+VITE_APP_AWS_COOKIE_LOCATION="localhost"
+ASSET_URL="http://localhost:3000"
```
Comment (robisonmark) [inline .env:5]: I don't see this else where, is it needed? I don't remember the code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  <meta name="msapplication-TileColor" content="#2784C3">
+  <meta name="theme-color" content="#2784C3">
+
+  <!-- Vue MultiSelect Default styles -->
+  <style src="vue-multiselect/dist/vue-multiselect.min.css"></style>
+
+  <title>MHAC | Midsouth Homeschool Athletics</title>
+
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157989243-1"></script>
```
Comment (robisonmark) [inline index.html:25]: I don't know if this has been updated on this branch but I have another branch that updates the Google Analytics

File: index.html
Comment (robisonmark) [inline index.html:1]: What's this index?

File: src/components/editTable.vue [hunk 3]
```diff
@@ -15,7 +15,7 @@
       <slot name="tbody">
         <tr v-for="(data, index) in tabledata" :key="index">
           <template v-for="(column, idx) in columns">
-            <td v-if="!column.field_name.includes('id')" :key="column.field_name + idx" :class="idx">
```
Comment (robisonmark) [inline src/components/editTable.vue:18]: I see a lot of removing the key, is this recommended from vue. I know in vue 2 the key was used for tracking updates so as to not re-render more than necessary.

**Review by robisonmark — APPROVED** [review]
Left a couple of concerns but looks to be similar on each file so go ahead and merge

File: index.html (hunk 2 above)
Comment (mjrobison) [reply index.html:25]: I didn't update anything here, but it will end up with a problem, this file had to change location in Vue3

File: index.html
Comment (mjrobison) [reply index.html:1]: Vue3 Change used to be in public.

**Pull request merged**